app/services/tsheets_service.rb

```diff
@@ -0,0 +1,29 @@
+class TsheetsService
```

Comment:
I'm not sure, what is a difference TsheetsService from TsheetsAdapter? Why can't find_users be in this service class? Do we have any other adapters/or need for them now?

It seems, that Tsheet as a losy abstraction  - I remember one of Sandy Metz talks, then she was warning against doing too much abstractions without a need. I would be happy if you could drop it, it seems a bit confusing.

I also partly understand why you added adapter object - it seems like you want to encapsulate all calls to Tsheets. if that is the case, then maybe adapter is not a project name? It's more of wrapper? maybe let's move it to `lib/` then?